README.md

```diff
@@ -119,23 +120,29 @@ if err != nil {
 }
 ```
 
-Retrieve measure by name:
+### Recording Measurements
 
-[embedmd]:# (stats.go findMeasure)
+Measurements are data points associated with Measures.
+Recording implicitly tags the set of Measurements with the tags from the
```

Comment:
Hmm, given this mention here, perhaps it would help if in the example we inserted a tag into a context right before recording.

Reply:
There's a separate section on tags. This statement just says why we take a context parameter to Record.

Reply:
Roger that, thanks for the clarification.